src/labels/CustomLabels.labels

```diff
@@ -6670,6 +6670,118 @@ If preferred, you can disable Gift Entry and instead use older Batch Gift Entry
+        <fullName>gsChecklistItemExtraOneMinuteVideo</fullName>
+        <categories>Get Started Page</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Extra Info in the Item &apos;1 Minute Video&apos;</shortDescription>
+        <value>1 Minute Video</value>
+    </labels>
+    <labels>
+        <fullName>gsChecklistItemSetupContactDesc</fullName>
+        <categories>Get Started Page</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Description of checklist item Setup Contact in Get Started</shortDescription>
+        <value>People are the most important resource for a nonprofit, so understanding the Contact Object is especially important. Learn how to add relevant constituent data (e.g. a name, address, or employer) into your Salesforce instance.</value>
+    </labels>
+    <labels>
+        <fullName>gsChecklistItemSetupContactPriBtnLabel</fullName>
+        <categories>Get Started Page</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Button label of checklist item Setup Contact in Get Started</shortDescription>
+        <value>Create Household</value>
+    </labels>
+     <labels>
+        <fullName>gsChecklistItemSetupContactLinkLabel</fullName>
+        <categories>Get Started Page</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Link label of checklist item Setup Contact in Get Started</shortDescription>
+        <value>Learn more on Trailhead</value>
+    </labels>
+    <labels>
+        <fullName>gsChecklistItemSetupContactSecBtnLabel</fullName>
+        <categories>Get Started Page</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Button label of checklist item Setup Contact in Get Started</shortDescription>
+        <value>Explore Contacts</value>
+    </labels>
+    <labels>
+        <fullName>gsChecklistItemSetupContactTitle</fullName>
+        <categories>Get Started Page</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Title of checklist item Setup Contact in Get Started</shortDescription>
+        <value>Setup Contact and Household</value>
+    </labels>
+    <labels>
+        <fullName>gsChecklistItemStayTrackDesc</fullName>
+        <categories>Get Started Page</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Description of checklist item Stay Track in Get Started</shortDescription>
+        <value>&quot;Activities“ refers to tasks, events, or any other interaction that’s tracked in the Activity Timeline. Understanding these will allow you to keep track of every interaction with your constituents. Learn how to leverage activities to log constituent interactions and assign tasks.</value>
```

Comment:
Please change this text to "Activities are tasks, events, or other interactions tracked in the Activity Timelines. Activities allow you to keep track of each time you have contact with a constituent. Learn how to leverage activities to log constituent interactions and assign tasks. "